packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/zoe-full-upgrade/zcf-upgrade-driver.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+. ./upgrade-test-scripts/env_setup.sh
+
+set -euo pipefail
+
+# This shows how to upgrade Zoe and ZCF on a running chain. It presumes that
+# The bundles for Zoe and ZCF have been installed, and their hashes updated in
+# zcf-upgrade-script.js. The bundles can be generated by running the test in
+# vats/test/bootstrapTests/test-zcf-upgrade.js, noting the logged ZOE BUNDLE ID
+# and ZCF BUNDLE ID, and finding the corresponding bundles in ~/.agoric/cache.
+#  !!   THERE HAS TO BE A BETTER WAY   !!
```

Comment:
there will be! we just have to design and implement it :)

Reply:
There are slightly clearer instructions now in actions.sh.